### PR TITLE
handle binding patterns correctly when getting script lexical structure

### LINKED ISSUE #1791: error from 'getNavigationBarItems' when using binding patterns in variable declarations
Repro code:

```typescript
var a, {}
```

### PR DESCRIPTION
**Pull request by vladima**
Fixes #1791

### CODE REVIEW

**Comment by DanielRosenwasser** [comment]
Vlad, this looks too simple to work.

:+1:

File: tests/cases/fourslash/scriptLexicalStructureBindingPatterns.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/// <reference path='fourslash.ts'/> 
+////var foo, {}
+////var bar, []
+verify.getScriptLexicalStructureListCount(3); // global (1) + variable declarations (2)
```
Comment (JsonFreeman) [inline tests/cases/fourslash/scriptLexicalStructureBindingPatterns.ts]: If the binding patterns have names inside, do we show those as nav bar items? Can you add a test for that?

Comment (vladima) [inline tests/cases/fourslash/scriptLexicalStructureBindingPatterns.ts]: sure

**Comment by sheetalkamat** [comment]
:+1:

**Comment by JsonFreeman** [comment]
:+1:

**Pull request merged**